socorro/external/es/crashstorage.py

```diff
@@ -211,6 +211,77 @@ def save_raw_and_processed(self, raw_crash, dumps, processed_crash,
 
 
 #==============================================================================
+class ESCrashStorageRestrictedDump(ESCrashStorage):
```

Comment:
s/Restricted/Redacted

To be "restricted" means when your covering or show less. But what it does is that it doesn't just show you a restricted crash, it modifies it. 

Also, "Dump"? To me, in socorro, "dump" refers to the binary blobs and not around the JSON crashes.